chapters/peas-and-pods.rst

```diff
@@ -0,0 +1,155 @@
+Development Guide: Peas and Pods in Jina
+=========================================
+
+.. meta::
+   :description: Development Guide: Peas and Pods in Jina
+   :keywords: Jina, pea, pod
+
+.. note:: This guide assumes you have a basic understanding of Jina, if you haven't, please check out `Jina 101 <https://101.jina.ai>`_ first.
+.. note:: This guide assumes you have a basic understanding of Parallelization inside Jina, if you haven't, please check out `Parallelization in Jina <../parallel>`_ first.
+
+You might have already learned from `Jina 101 <../101.rst>`_.
+Jina :term:`Pea` wraps an :term:`Executor` and lets it exchange data with other Peas.
+Jina :term:`Pod` is a context manager for one or multiple Peas that have the same properties.
```

Comment:
I would add something on these lines.

A Pod is an abstraction layer on top of Peas equipped with the same Executors allowing the user to reason about the application and design its functionality hiding the parallelization and the non-functional details

Reply:
added @JoanFM